test/config/FiltersSpec.scala

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
personally i don't care about the tests for this non-prod service  -  
if we want tests, then make sure the min coverage is set to atleast `90` not `80` ?